variant/data_dictionary.json

```diff
@@ -69,5 +69,6 @@
     "primitive_string": "This string is longer than 64 bytes and therefore does not fit in a short_string and it also includes several non ascii characters such as \ud83d\udc22, \ud83d\udc96, \u2665\ufe0f, \ud83c\udfa3 and \ud83e\udd26!!",
     "primitive_timestamp": "2025-04-16 12:34:56.78-04:00",
     "primitive_timestampntz": "2025-04-16 12:34:56.78",
-    "short_string": "Less than 64 bytes (\u2764\ufe0f with utf8)"
-}
\ No newline at end of file
+    "short_string": "Less than 64 bytes (\u2764\ufe0f with utf8)",
+    "primitive_time": "12:33:54:123456"
```

Comment:
The value copied from [iceberg test](https://github.com/apache/iceberg/blob/3a4215dbb714477c89681ab94f1197b6ebcbdfff/parquet/src/test/java/org/apache/iceberg/parquet/TestVariantReaders.java#L133)